jupyter-book/introduction/analysis_tools.ipynb

```diff
@@ -52,9 +52,16 @@
    "cell_type": "markdown",
```

Comment:
<p>Using <code>ad</code> as a variable name here may not be a great recommendation for newcomers since it's already used for the <code>anndata</code>  module.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='145'/>